recipe/meta.yaml

```diff
@@ -47,8 +47,7 @@ outputs:
         - {{ pin_subpackage("arrow-cpp", max_pin="x.x.x") }}
       ignore_run_exports:
         - cudatoolkit
-      track_features:
-        {{ "- arrow-cuda" if cuda_enabled else "" }}
+      track_features: {{ "- arrow-cuda" if cuda_enabled else "" }}
```

Comment:
FYI this update bot change appears to be breaking re-rendering

cc @conda-forge/bot

Reply:
Filed as issue ( https://github.com/regro/cf-scripts/issues/1419 )

Reply:
The recipe has to be changed here if you want to use the bot. This specific syntax is too fragile.

Reply:
See my suggestions in the issue above.

Reply:
Done in the latest commit